Cargo.toml

```diff
@@ -82,3 +82,7 @@ path = "examples/skybox/skybox.rs"
 [[example]]
 name = "ui"
 path = "examples/ui/ui.rs"
+
+[[example]]
+name = "egui"
+path = "examples/egui/main.rs"
```

Comment:
We should either rename it into `egui.rs` or all other examples to `main.rs`
Is there a reason, why you went with `main.rs`?

Reply:
Yes, you're right. I don't remember, I propably went for `main.rs` because it is mostly used for default executable in rust